src/client/views/book/bookEditor.ml

```diff
@@ -93,3 +117,13 @@ let submit t =
   let modified_at = Datetime.now () in
   let created_at = Datetime.now () in
   Book.make_and_save ~title ?date ~contents_and_parameters ~modified_at ~created_at ()
+
+let update_submit t slug =
+  let%lwt book = Book.get slug in
+  let title = t.title in
+  let date = if t.date <> "" then Some (PartialDate.from_string t.date) else None in
+  let contents = fold t (fun _ set acc -> snd set :: acc) [] in
+  let contents_and_parameters = List.map (fun set -> Book.Set (set, SetParameters.none)) contents in
+  let modified_at = Datetime.now () in
+  let%lwt created_at = Book.created_at book in 
+  Book.update ~slug ~title ?date ~contents_and_parameters ~modified_at ~created_at ()
```

Comment:
Should `update` take a `created_at` argument? This is a broader discussion out of scope for this PR, but I think it could be argued that `created_at` should be frozen after initialization.

Reply:
This is a good point.

Reply:
cf #208